### PR TITLE
feat(CFW): add cfw lts log resource

### PR DESCRIPTION
**Pull request by profoundwu**
**What this PR does / why we need it**:
add cfw lts log resource
**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
1. add cfw lts log resource
```

## PR Checklist

* [x] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.

## Acceptance Steps Performed

```
make testacc TEST="./huaweicloud/services/acceptance/cfw" TESTARGS="-run TestAccLtsLog_basic"
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/cfw -v -run TestAccLtsLog_basic -timeout 360m -parallel 4
=== RUN   TestAccLtsLog_basic
=== PAUSE TestAccLtsLog_basic
=== CONT  TestAccLtsLog_basic
--- PASS: TestAccLtsLog_basic (98.03s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/cfw       98.073s
```

### CODE REVIEW

File: docs/resources/cfw_lts_log.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  If omitted, the provider-level region will be used.
+  Changing this creates a new resource.
+
+* `fw_instance_id` - (Required, String, ForceNew) Specifies the ID of the firewall.
+  Changing this creates a new resource.
+
+* `lts_log_group_id` - (Required, String) Specifies the LTS log group ID.
+
+* `lts_attack_log_stream_enable` - (Required, Int) Specifies whether to enable the attack log stream.
+  The valid values are 0 and 1, where 0 means disable and 1 means enable.
```
Comment (houpeng80) [inline docs/resources/cfw_lts_log.md]:
```suggestion
* `lts_attack_log_stream_enable` - (Required, Int) Specifies whether to enable the attack log stream.
  The valid values are **0** and **1**, where **0** means disable and **1** means enable.
```
and fix other places

File: huaweicloud/services/acceptance/cfw/resource_huaweicloud_cfw_lts_log_test.go [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+	path := client.Endpoint + httpUrl
+	path = strings.ReplaceAll(path, "{project_id}", client.ProjectID)
+	path += fmt.Sprintf("?fw_instance_id=%s", state.Primary.ID)
+
+	opt := golangsdk.RequestOpts{
+		KeepResponseBody: true,
+	}
+	resp, err := client.Request("GET", path, &opt)
+	if err != nil {
+		return nil, fmt.Errorf("error retrieving lts log configuration: %s", err)
```
Comment (houpeng80) [inline huaweicloud/services/acceptance/cfw/resource_huaweicloud_cfw_lts_log_test.go]:
```suggestion
		return nil, fmt.Errorf("error retrieving CFW lts log configuration: %s", err)
```
and fix other places

File: huaweicloud/services/cfw/resource_huaweicloud_cfw_lts_log.go [hunk 3]
```diff
@@ -0,0 +1,332 @@
...
+
+	path := client.Endpoint + httpUrl
+	path = strings.ReplaceAll(path, "{project_id}", client.ProjectID)
+	path += fmt.Sprintf("?fw_instance_id=%s", d.Get("fw_instance_id"))
+
+	opt := golangsdk.RequestOpts{
+		KeepResponseBody: true,
+		OkCodes: []int{
+			200,
+		},
```
Comment (houpeng80) [inline huaweicloud/services/cfw/resource_huaweicloud_cfw_lts_log.go]: it is better to remove the default ok codes, and fix other places

File: huaweicloud/services/cfw/resource_huaweicloud_cfw_lts_log.go [hunk 4]
```diff
@@ -0,0 +1,332 @@
...
+		if errorCodeErr != nil {
+			return err
+		}
+
+		if errorCode == "CFW.00200005" {
+			return golangsdk.ErrDefault404(errCode)
+		}
+	}
+	return err
+}
```
Comment (houpeng80) [inline huaweicloud/services/cfw/resource_huaweicloud_cfw_lts_log.go]: this func can be merged with func parseCreateError

File: huaweicloud/services/cfw/resource_huaweicloud_cfw_lts_log.go (hunk 3 above)
Comment (profoundwu) [reply huaweicloud/services/cfw/resource_huaweicloud_cfw_lts_log.go]: fixed

File: huaweicloud/services/acceptance/cfw/resource_huaweicloud_cfw_lts_log_test.go (hunk 2 above)
Comment (profoundwu) [reply huaweicloud/services/acceptance/cfw/resource_huaweicloud_cfw_lts_log_test.go]: fixed

File: docs/resources/cfw_lts_log.md (hunk 1 above)
Comment (profoundwu) [reply docs/resources/cfw_lts_log.md]: fixed

File: huaweicloud/services/cfw/resource_huaweicloud_cfw_lts_log.go (hunk 4 above)
Comment (profoundwu) [reply huaweicloud/services/cfw/resource_huaweicloud_cfw_lts_log.go]: fixed

File: docs/resources/cfw_lts_log.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+
+* `lts_access_log_stream_id` - (Optional, String) Specifies the access log stream ID.
+
+* `lts_flow_log_stream_id` - (Optional, String) Specifies the flow log stream ID.
+
+## Attribute Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - The resource ID.
```
Comment (Jason-Zhang9309) [inline docs/resources/cfw_lts_log.md]:
```suggestion
* `id` - The resource ID. The value is the firewall instance ID.
```

File: docs/resources/cfw_lts_log.md [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+
+## Import
+
+The lts log resource can be imported using `id`, e.g.
+
+```bash
+$ terraform import huaweicloud_cfw_lts_log.test <id>
+```
```
Comment (Jason-Zhang9309) [inline docs/resources/cfw_lts_log.md]:
```suggestion
The lts log resource can be imported using the firewall instance ID, e.g.

```bash
$ terraform import huaweicloud_cfw_lts_log.test <fw_instance_id>
```
```

File: huaweicloud/services/acceptance/cfw/resource_huaweicloud_cfw_lts_log_test.go [hunk 7]
```diff
@@ -0,0 +1,173 @@
...
+	return fmt.Sprintf(`
+resource "huaweicloud_lts_group" "g1" {
+  group_name  = "%[1]s"
+  ttl_in_days = 1
+}
+
+resource "huaweicloud_lts_group" "g2" {
+  group_name  = "%[1]s_2"
+  ttl_in_days = 1
+}
```
Comment (Jason-Zhang9309) [inline huaweicloud/services/acceptance/cfw/resource_huaweicloud_cfw_lts_log_test.go:151]:
```suggestion
resource "huaweicloud_lts_group" "g1" {
  count = 2

  group_name  = "%[1]s_${count.index}"
  ttl_in_days = 1
}
```

**Pull request merged**